### PR TITLE
[ZT] apply content strategy

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

**Comment by paul1974nsac** [comment]
> _ No description provided. _

**Comment by paul1974nsac** [comment]
> > _ No description provided. _

File: content/cloudflare-one/_index.md [hunk 1]
```diff
@@ -32,6 +32,6 @@ Cloudflare Zero Trust replaces legacy security perimeters with our global edge,
 * Mitigate the impact of attacks
 * Seamless, lightning-fast end user experience
 
-(/cloudflare-one/static/documentation/teams-no-background.png)
+![](/cloudflare-one/static/documentation/teams-no-background.png)
```
Comment (abracchi-tw) [inline content/cloudflare-one/_index.md:35]: curious as to why we're not adding any alt text here?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md [hunk 2]
```diff
@@ -55,120 +52,37 @@ The format of this file is as follows:
...
-    <string>warp</string>
-    <key>support_url</key>
-    <string>https://support.example.com</string>
-</dict>
-</plist>
-```
+### Create `plist` file
 
-Here is an example `.mobileconfig` file with the accepted arguments:
+1. [Click here](/cloudflare-one/static/documentation/connections/com.cloudflare.warp.plist) to download an example `com.cloudflare.warp.plist` file.
```
Comment (abracchi-tw) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]:
```suggestion
1. [Download](/cloudflare-one/static/documentation/connections/com.cloudflare.warp.plist) an example `com.cloudflare.warp.plist` file.
```

(avoiding `click here` in hyperlinks)

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md [hunk 3]
```diff
@@ -55,120 +52,37 @@ The format of this file is as follows:
...
-2.  Run the following command:
+        ```txt
+        % plutil -convert binary1 com.cloudflare.warp.plist
+        ```
 
-<!---->
+### Create `mobileconfig` file
 
-    % plutil -convert binary1 com.cloudflare.warp.plist
+1. [Click here](/cloudflare-one/static/documentation/connections/CloudflareWARP.mobileconfig) to download an example `.mobileconfig` file.
```
Comment (abracchi-tw) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]: same as above

**Review by abracchi-tw — APPROVED** [review]

File: content/cloudflare-one/_index.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/_index.md:35]: I read that decorative images should not have alt text.
https://wiki.cfops.it/display/Prod/Alt+text+guidelines#Alttextguidelines-Whichimagesneedalttext?

File: content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md [hunk 4]
```diff
@@ -55,120 +52,37 @@ The format of this file is as follows:
...
```
Comment (ranbel) [inline content/cloudflare-one/connections/connect-devices/warp/deployment/mdm-deployment/_index.md]:
```suggestion
1. [Download](/cloudflare-one/static/documentation/connections/CloudflareWARP.mobileconfig) an example `.mobileconfig` file.
```

**Review by phuho2020 — APPROVED** [review]

**Comment by David12345678912** [comment]
https://github.com/cloudflare/cloudflare-docs/pull/4103/commits/4a6f95983496c4f1ed999f12d17a01a9fbd850be

**Pull request merged**